chapters/tr/_toctree.yml

```diff
@@ -17,3 +16,20 @@
   sections:
   - local: chapter2/1
     title: Giriş
+
+- title: 3. Pretrained bir modeli fine-tune etmek
+  sections:
+  - local: chapter3/1
+    title: Giriş
+  - local: chapter3/2
```

Comment:
We should only include the sections here that have already been translated, so everything from here and below can be removed for now :)

Reply:
i think its done.